Mathlib/RingTheory/KrullDimension/NonZeroDivisors.lean

```diff
@@ -18,24 +18,23 @@ import Mathlib.RingTheory.Spectrum.Prime.Topology
 - `ringKrullDim_add_enatCard_le_ringKrullDim_mvPolynomial`: `dim R + #σ ≤ dim R[σ]`.
 -/
 
-open nonZeroDivisors
-
 variable {R S : Type*} [CommRing R] [CommRing S]
 
-lemma ringKrullDim_quotient (I : Ideal R) :
-    ringKrullDim (R ⧸ I) = Order.krullDim (PrimeSpectrum.zeroLocus (R := R) I) := by
-  let e : PrimeSpectrum (R ⧸ I) ≃ (PrimeSpectrum.zeroLocus (R := R) I) :=
-    (Equiv.ofInjective _ (PrimeSpectrum.comap_injective_of_surjective _
-      Ideal.Quotient.mk_surjective)).trans (Equiv.setCongr
+/-- The prime spectrum of a quotient of a ring by an ideal is in order-preserving bijection
+with the zero locus of the ideal. -/
+noncomputable def PrimeSpectrum.quotientOrderIso (I : Ideal R) :
+    PrimeSpectrum (R ⧸ I) ≃o PrimeSpectrum.zeroLocus (R := R) I where
+  toEquiv := (Equiv.ofInjective _ (PrimeSpectrum.comap_injective_of_surjective _
```

Comment:
Note: use [this](https://github.com/leanprover-community/mathlib4/pull/25210/files#diff-14e0882f79f4ea69a6b16a99b513228a4e2a5bdd57b7d9dc11ac8e55c935b1caR247) once #25210 is merged.

Reply:
Another TODO: replace [these two](https://github.com/leanprover-community/mathlib4/pull/25210/files#diff-5a42781dcb3443ee485415566fc91a414e007300faef6f2ef8de56cea71c7f6cR295-R314) with my version

Reply:
Maybe also fix indentation [here](https://github.com/leanprover-community/mathlib4/pull/25210/files#diff-5cdb1b6453954cff9be1fd36ae685ad335238430e72c3d56d0e786e23922960aR30) and below :)